bagelbids/cli.py

```diff
@@ -1,75 +1,26 @@
 from pathlib import Path
-from typing import Dict
-import warnings
 
-import click
-from bids import BIDSLayout
+import typer
 
-from bagelbids import models, mappings
+bagel = typer.Typer()
 
 
-def map_term_to_namespace(term: str, namespace: Dict):
+@bagel.command()
+def pheno(
+        pheno: Path = typer.Option(..., help="The path to a demographic .tsv file.",
```

Comment:
Minor but should we stick to calling these "phenotypic" rather than "demographic" files in anticipation of them containing also assessment data? I think the various types of data files may get confusing quickly so I'm eager to keep things consistent where we can :)

Reply:
Oh, good catch. You're referring to the help text here? I'll change that!